src/s2/base/commandlineflags.h

```diff
@@ -52,35 +45,24 @@ inline void SetFlag(google::int64* f, google::int64 v) { *f = v; }
 
 #else  // !defined(S2_USE_GFLAGS)
 
-#include <string>
-
-#include "s2/base/integral_types.h"
 
 // Create a set of gflags-like macros for declaring/defining flags. Use
 // a library-specific name to potential minimize clashes with GFlags.
 
 #define S2_DEFINE_bool(name, default_value, description) \
   bool FLAGS_##name = default_value
-#define S2_DECLARE_bool(name) \
```

Comment:
I don't understand, branch doesn't compile without it

Reply:
Looks like commandlineflags_declare.h is broken.

https://github.com/google/s2geometry/pull/297/files/be8a56d2952660b3c977a0a70ba861e597dbe9e0#r1065513981

Reply:
Fixed.